train.py

```diff
@@ -26,9 +28,14 @@
 # Parameters
 # ----------------------------------------------------------------------------------------------------------------------
 DEVICE = torch.device("cuda" if torch.cuda.is_available() else "cpu")
-competition = "ref_fusion_competition_south_africa"
+
 arg_parser = ArgumentParser()
-arg_parser.add_argument("--competition", type=str, default=competition)
+arg_parser.add_argument(
+    "--competition", 
+    type=str, 
+    default="germany",
+    help="germany, south_africa"
+)
```

Comment:
If the original wandb project is for south africa then it would make sense for the default competition to be south africa too probably